components/features/Dashboard/Dashboard.test.tsx

```diff
@@ -13,12 +13,13 @@ describe("Dashboard test suite", () => {
         expect(taskbarEl).toHaveAttribute("class", "taskbar")
     })
     test("testing open app", () => {
-        const { getByTestId } = render(<Dashboard />);
+        const { getByTestId,  } = render(<Dashboard />);
```

Comment:
Hello there Rashad, is there another element you want to get from the following statement?

Reply:
okay Alex, I've fixed the issue👌